api/models/uaa-identity.js

```diff
@@ -0,0 +1,56 @@
+const associate = ({ UAAIdentity, User }) => {
+  UAAIdentity.belongsTo(User, {
+    foreignKey: 'userId',
+    allowNull: false,
+  });
+};
+
+module.exports = (sequelize, DataTypes) => {
+  const UAAIdentity = sequelize.define('UAAIdentity', {
+    uaaId: {
+      type: DataTypes.STRING,
+      unique: true,
+      validate: {
```

Comment:
I thought we'll have the uaa id when we create this record so can't we make this not nullable?

Reply:
I had it at non nullable but changed due to not wanting to create placeholder values in the local dev seed data. It think I'll add it back for integrity.

Reply:
Updated